vnext/ReactUWP/Base/UwpReactInstance.cpp

```diff
@@ -254,7 +261,7 @@ void UwpReactInstance::Start(const std::shared_ptr<IReactInstance>& spThis, cons
     m_uiManager = CreateUIManager(spThis, m_viewManagerProvider);
 
     // Acquire default modules and then populate with custom modules
-    std::vector<facebook::react::NativeModuleDescription> cxxModules = GetModules(m_uiManager, m_defaultNativeThread, deviceInfo, devSettings, std::move(i18nInfo), std::move(appstate));
+    std::vector<facebook::react::NativeModuleDescription> cxxModules = GetModules(m_uiManager, m_defaultNativeThread, deviceInfo, devSettings, std::move(i18nInfo), std::move(appstate), spThis);
```

Comment:
>spThis [](start = 185, length = 6)

I'd like you pass weak_ptr to the callback if possible unless you know the lifecycle of spThis very well.
Otherwise it's possible to introduce reference cycle.